src/main/java/net/dv8tion/jda/internal/handle/GuildMemberUpdateHandler.java

```diff
@@ -117,6 +120,17 @@ protected Long handleInternally(DataObject content)
                                 member, oldNick));
             }
         }
+        if (content.hasKey("premium_since"))
+        {
+            long boostDate = 0;
+            if (!content.isNull("premium_since"))
```

Comment:
Forgot the else here which sets it to 0 and removing the boostDate variable which is unused